src/dispatch/incident/messaging.py

```diff
@@ -591,3 +593,32 @@ def send_incident_resources_ephemeral_message_to_participant(
     )
 
     log.debug(f"List of incident resources sent to {user_id} via ephemeral message.")
+
+
+def send_incident_close_reminder(incident: Incident):
+    """Sends a direct message to the incident reminding them to close the incident if possible."""
+    message_text = f"Incident Status Reminder"
+    message_template = INCIDENT_STATUS_REMINDER
+
+    convo_plugin = plugins.get(INCIDENT_PLUGIN_CONVERSATION_SLUG)
+    update_command = convo_plugin.get_command_name(ConversationCommands.edit_incident)
```

Comment:
nit: we should change the name of the enum to `update_incident` to be consistent with the command name.

Reply:
Yeah, I had `update_incident` first until I realized that wasn't the enum.

Reply:
Yeah, looks like we never updated the enum when we renamed the command from edit to update.

Reply:
Don't forget to update it here as well: https://github.com/Netflix/dispatch/blob/develop/src/dispatch/conversation/enums.py#L6